portfolio/src/main/webapp/script.js

```diff
@@ -109,7 +135,7 @@ async function changeName() {
     params.append('nickname', nickname);
     let response = await fetch('/auth', {method: 'POST', body: params});
     if (!response.ok) {
-        alert('HTTP-Error: ' + response.status);
+    	alert('HTTP-Error: ' + response.status);
```

Comment:
```suggestion
		alert('HTTP-Error: ' + response.status);
```
seems like this one is using spaces instead of tab :P